src/components/mapexplorer.js

```diff
@@ -380,12 +382,34 @@ function MapExplorer({
 
   const {name, lastupdatedtime} = currentHoveredRegion;
 
+  function getCurrentDate() {
+    const date = new Date();
+    let getMonth = String(date.getMonth() + 1);
+    if (getMonth.length === 1) {
+      getMonth = '0' + getMonth;
+    }
+    const dateComp = date.getDate() + '/' + getMonth + '/' + date.getFullYear();
+    return dateComp;
+  }
+
   useEffect(() => {
+    const testDataFind = stateTestData.filter(
+      (obj) => obj.totaltested !== '' && obj.updatedon === getCurrentDate()
+    );
+
+    const totalTMap = testDataFind.map((value) => value.totaltested);
+    totalTMap.forEach((element) => {
+      // eslint-disable-next-line
+      totalTemp += Number(element);
+    });
+
+    setTotal(totalTemp);
```

Comment:
remove the extra variable 'totalTemp'. You could use, js reduce function to setTotal. This way you would not be required to disable eslint and code will be more concise and readable.

Reply:
This was my second attempt at React programming.

I knew I was missing state management of some sort.

Thanks for input. I’ll check & update the codebase.